exit status: 0
stdout:
let x : Bool = (true, false, true)._0;
let y : Bool = (true, false, true)._1;
() : ()

stderr:
error: duplicate definition of local name `x`
  ┌─ tests/fail/match/duplicate-name.pion:4:12
  │
4 │ let (x, y, x) = (true, false, true);
  │      -     ^ duplicate definition
  │      │      
  │      first definition

